app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The function `count_occurrences` is supposed to return an integer representing the number of occurrences of `letter` in `phrase`. However, the `count` method already returns an integer, so there is no need for further modification. Ensure that the function returns the correct type as expected by the task requirements.